ui/src/index.html

```diff
@@ -114,6 +114,10 @@ <h2>How have you been at Zenika?</h2>
             <span class="button__emoji">😤</span> Not great at all
           </button>
         </div>
+        <div>
+          <h2>A litte comment?</h2>
+          <textarea id="comment" rows="10"></textarea>
```

Comment:
30 rows doesn't seem much, maybe adapt it to its container ?

Reply:
![Capture d’écran_2019-04-25_10-03-23](https://user-images.githubusercontent.com/42938784/56719628-6cf45d80-6741-11e9-99b7-e97a3be1d2e7.png)

I think it's okay for the numbers of rows ?

But i'm not sure about the vote workflow :/ The user must write a comment before the click.